packages/strapi-generate-new/lib/before.js

```diff
@@ -192,7 +192,7 @@ module.exports = (scope, cb) => {
               ])
               .then(answers => {
                 if (hasDatabaseConfig) {
-                  answers = _.omit(scope.database.settings, ['client']);
+                  answers = _.merge((_.omit(scope.database.settings, ['client'])), scope.database.options);
```

Comment:
It seems I had to merge the optional options to the final `answers` object. Let me know if this makes sense.